data/json/vehicleparts/boards.json

```diff
@@ -132,6 +133,7 @@
     "name": "extra light quarterpanel",
     "item": "pipe",
     "location": "center",
+    "looks_like": "halfboard",
```

Comment:
this works properly with abstract?

Reply:
Looking further, it's probably not needed in abstract even if it does something.